City-of-Pori/pori-kada
# PORI-190 accessibility icons

To test:

1. drush fra -y && drush cc all
2. Go to edit the accessibility taxonomy at /admin/structure/taxonomy/accessibility
3. Edit each term and add them name, class that matches that name and add description to some of the terms as well (notice the help text on description).
4. Add terms that are not present in the database yet: Free wifi, Free parking, Pets allowed and Allergy friendly.
5. Go to edit any attraction card and add the accessibility options that you want, maybe all of them to make sure they all work.
6. View the attraction card and make sure all the accessibility options have icons and the ones that have description are clickable and show the description when clicked. Also make sure the description is closed when you click away from the description or click the description again.
